pulsar-broker/src/test/java/org/apache/pulsar/io/PulsarSinkE2ETest.java

```diff
@@ -189,6 +189,11 @@ public void testPulsarSinkDLQ() throws Exception {
             remainingMessagesToReceive.add(messageBody);
         }
 
+        // Wait all msg show in dlqTopic.
+        Awaitility.await().untilAsserted(()-> {
+            assertEquals(admin.topics().getLastMessageId(dlqTopic).toString(),"10:1:-1:8");
```

Comment:
Can we guarantee that the message-id is the same for every testing? There is an exception in this PR:https://github.com/apache/pulsar/runs/7524494144?check_suite_focus=true#step:10:1231

We can directly modify the wait time for the consumer to receive the message:
https://github.com/apache/pulsar/pull/16799/files#diff-1be1a68c630a114a3f7770115053f58066cbf117b24d0e73903f6340f7aef793L194

And I think the root cause may not be the timeout setting. 10 seconds should be enough.

Reply:
> Can we guarantee that the message-id is the same for every testing? There is an exception in this

seems that you are right